logstash_async/constants.py

```diff
@@ -32,11 +32,11 @@ class Constants:
     # Usually this list does not need to be modified. Add/Remove elements to
     # exclude/include them in the Logstash event, for the full list see:
     # http://docs.python.org/library/logging.html#logrecord-attributes
-    FORMATTER_RECORD_FIELD_SKIP_LIST = [
+    FORMATTER_RECORD_FIELD_SKIP_LIST = {
```

Comment:
Cool, I wasn't aware there is such a difference on lookup performance between lists and sets.

It would be even better if we keep the constant a list here and convert it to a set later in the Formatter class, probably best in its `__init__()`. Then people can still append/remove items as they wish. Even if named constants, they are meant to be customised if necessary.
A set is also mutable but with a different API and so users had to adjust their code.

I think converting the list to a set later when creating a Formatter instance, should be ok as this happens only once (or only a few times if the formatter is configured multiple times or on forking or so but still not as often as log records are to be formatted).